tools/modm_tools/elf2uf2.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2021, Andrey Kunitsyn
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Authors:
+# - 2022, Andrey Kunitsyn
+
```

Comment:
```
"""
### ELF to UF2 Converter

describe what it does
"""
```

This documentation get's picked up by the docs generator and put onto the homepage: https://modm.io/reference/module/modm-build/#generic-python-tools